internal/output/html/package_table_template.html

```diff
@@ -1,53 +1,62 @@
-      {{ $element.FixedVersion }} (fixes {{ $element.HTMLVulnCount.Fixed }}/{{ $element.HTMLVulnCount.Called }})
+    <td {{ if eq $element.HTMLVulnCount.Called 0 }}class="uncalled-text" {{ end }}>{{ $element.Name }}</td>
+    <td {{ if eq $element.HTMLVulnCount.Called 0 }}class="uncalled-text" {{ end }}>{{ $element.InstalledVersion }}</td>
+    <td>
+      {{ if ne $element.HTMLVulnCount.UnFixed 0 }}
+      {{ if eq $element.HTMLVulnCount.Fixed 0}}
+      <p class="fixable-tag no-fix">No fix available</p>
+      {{ else }}
+      <div class="tooltip">
+        <p class="fixable-tag has-fix">Partial fixes available</p>
+        <span class="tooltiptext">Upgrading the package will fix {{ $element.HTMLVulnCount.Fixed }} out of {{
+          $element.HTMLVulnCount.Called }} vulnerabilities</span>
+      </div>
+      {{ end }}
+      {{ else }}
+      <p class="fixable-tag has-fix">Fix available</p>
+      {{ end }}
     </td>
-    {{ end }}
-    {{ else }}
     <td>
-      {{ $element.FixedVersion }}
+      {{ if ne $element.HTMLVulnCount.Called 0 }}
+      {{ template "severity_summary_template.html" $element.HTMLVulnCount }}
+      {{ end }}
     </td>
-    {{ end }}
-    <td>{{ printSeverityCountShort $element.HTMLVulnCount }}</td>
   </tr>
   <tr>
     <td colspan="100%">
-      <div id="vuln-details-{{ $element.Name }}-{{ $rowIndex }}" class="vuln-details">
-        <p><strong>Source: {{ $element.Source }}</strong></p>
-        <p>Vulnerabilities:</p>
-        {{ range $index, $vuln := $element.CalledVulns }}
-        {{ if eq $vuln.Summary.FixedVersion "No fix available" }}
-        <p><a href="https://osv.dev/{{ $vuln.Summary.ID }}">{{ $vuln.Summary.ID }}</a>
-          ({{$vuln.Summary.SeverityRating}}) has no fix available</p>
-        {{ else }}
-        <p><a href="https://osv.dev/{{ $vuln.Summary.ID }}">{{ $vuln.Summary.ID }}</a>
-          ({{$vuln.Summary.SeverityRating}}) has a fixed version at {{$vuln.Summary.FixedVersion}}</p>
-        {{ end }}
+      <div id="table-tr-{{ $index }}-details" class="package-details">
+        {{ if and $element.HTMLPackageDetail (not (eq $element.HTMLPackageDetail.LayerID "")) }}
+        <div class="layer-command-container">
+          <span class="package-detail-title">Layer Command:</span>
+          {{ if ne $element.HTMLPackageDetail.LayerCommandTooltip "" }}
+          <div class="tooltip">{{ $element.HTMLPackageDetail.LayerCommand }}
+            <span class="tooltiptext">{{ $element.HTMLPackageDetail.LayerCommandTooltip }}</span>
+          </div>
+          {{ else }}
+          <div>{{ $element.HTMLPackageDetail.LayerCommand }}</div>
+          {{ end }}
+        </div>
+        <p><span class="package-detail-title">Layer ID:</span> {{ $element.HTMLPackageDetail.LayerID }}</p>
+        <p><span class="package-detail-title">In Base Image:</span> {{ $element.HTMLPackageDetail.InBaseImage }}</p>
         {{ end }}
-      </div>
+        {{ template "vuln_table_template.html" $element }}
+
```

Comment:
issue: you've mistakenly removed the closing `div`:

```suggestion
        {{ template "vuln_table_template.html" $element }}
      </div>
```